### ISSUE TITLE
Anaconda on Ubuntu 16.04 fails to link

### ISSUE THREAD

**Issue by ohlr** [Nuitka/Nuitka#436]
```
OS: Ubuntu 16.04
conda: 4.6.14
django: 2.2.2
Nuitka: 0.6.4 (installed through pip in a conda environment)
Python: 3.6.7

```
### use
`python -m nuitka --follow-imports --standalone hello_world.py`

### main
```
print("hello world")
```
Logs:
```
Nuitka:INFO:PASS 1:
Nuitka:INFO:Initial optimization pass.
Nuitka:INFO:Optimizing module '__main__', 0 more modules to go after that.
Nuitka:INFO:Doing module local optimizations for '__main__'.
Nuitka:INFO:Next global optimization pass.
Nuitka:INFO:Optimizing module '__main__', 0 more modules to go after that.
Nuitka:INFO:Doing module local optimizations for '__main__'.
Nuitka:INFO:PASS 2 ... :
Nuitka:INFO:Next global optimization pass.
Nuitka:INFO:Optimizing module '__main__', 0 more modules to go after that.
Nuitka:INFO:Doing module local optimizations for '__main__'.
Nuitka:INFO:Total memory usage before generating C code: 25.49 MB (26726400 bytes):
Nuitka:INFO:Included compiled module '__main__'.
Nuitka:INFO:Total memory usage before running scons: 25.74 MB (26992640 bytes):
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_abs':
(.text+0x75b): Nicht definierter Verweis auf `hypot'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_pow':
(.text.unlikely+0x3f6): Nicht definierter Verweis auf `hypot'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_pow':
(.text.unlikely+0x406): Nicht definierter Verweis auf `pow'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_pow':
(.text.unlikely+0x41d): Nicht definierter Verweis auf `atan2'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_pow':
(.text.unlikely+0x44f): Nicht definierter Verweis auf `exp'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_pow':
(.text.unlikely+0x468): Nicht definierter Verweis auf `log'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(complexobject.o): In Funktion `_Py_c_pow':
(.text.unlikely+0x494): Nicht definierter Verweis auf `sincos'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_rem':
(.text+0x32f): Nicht definierter Verweis auf `fmod'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_round.cold.11':
(.text.unlikely+0x468): Nicht definierter Verweis auf `round'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_pow.cold.13':
(.text.unlikely+0x4fb): Nicht definierter Verweis auf `fmod'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_divmod':
(.text.unlikely+0xc19): Nicht definierter Verweis auf `fmod'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_divmod':
(.text.unlikely+0xc91): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_is_integer':
(.text.unlikely+0x1552): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_round':
(.text.hot+0xcd4): Nicht definierter Verweis auf `round'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_pow':
(.text.hot+0x117e): Nicht definierter Verweis auf `pow'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_pow':
(.text.hot+0x1309): Nicht definierter Verweis auf `fmod'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(floatobject.o): In Funktion `float_pow':
(.text.hot+0x13c4): Nicht definierter Verweis auf `fmod'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(longobject.o): In Funktion `PyLong_FromString':
(.text.hot+0x4f65): Nicht definierter Verweis auf `log'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_Round':
(.text+0x104): Nicht definierter Verweis auf `ceil'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_Round':
(.text+0x11b): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_Round':
(.text+0x139): Nicht definierter Verweis auf `round'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_Round':
(.text+0x171): Nicht definierter Verweis auf `ceil'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_Round':
(.text+0x17f): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_Round.cold.8':
(.text.unlikely+0x237): Nicht definierter Verweis auf `round'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_FromSecondsObject':
(.text.hot+0x4c2): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_FromSecondsObject':
(.text.hot+0x58a): Nicht definierter Verweis auf `round'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_FromSecondsObject':
(.text.hot+0x5c7): Nicht definierter Verweis auf `ceil'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_FromSecondsObject':
(.text.hot+0x5d8): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(pytime.o): In Funktion `_PyTime_FromSecondsObject':
(.text.hot+0x5f1): Nicht definierter Verweis auf `round'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(_threadmodule.o): In Funktion `PyInit__thread':
(.text+0x8a): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(signalmodule.o): In Funktion `timeval_from_double':
(.text.hot+0x9b0): Nicht definierter Verweis auf `floor'
~/anaconda3/envs/myenv/lib/libpython3.6m.a(signalmodule.o): In Funktion `timeval_from_double':
(.text.hot+0x9cc): Nicht definierter Verweis auf `fmod'
collect2: error: ld returned 1 exit status
scons: *** [hello_world.bin] Error 1
```

Glad about any suggestions how to overcome this.

**Comment by kayhayen** [comment]
This seems to be a hello world issue.

Not sure what triggers this, probably a difference in what `glibc` and `libm` do.

**Comment by kayhayen** [comment]
So I made a guess fix on factory, looking at `--show-scons` for my anaconda, it may be the case that it should be later in the linker command line, which otherwise looks good. Please try it out, and let me know if it helps:

https://nuitka.net/doc/factory.html

**Comment by ohlr** [comment]
Yes, it runs through now

**Comment by kayhayen** [comment]
So this is in 0.6.5rc3, currently develop branch.

**Comment by kayhayen** [comment]
Released as 0.6.5
